backend/src/services/certificate-authority/certificate-authority-service.ts

```diff
@@ -1373,6 +1377,8 @@ export const certificateAuthorityServiceFactory = ({
       notAfterDate = new Date(notAfter);
     } else if (ttl) {
       notAfterDate = new Date(new Date().getTime() + ms(ttl));
+    } else if (certificateTemplate?.ttl) {
```

Comment:
Missing in `issueCertFromCa` method. Also, in the certificate template modal it states this field as **Max TTL** but this isn't enforced in this check; this seems to be more like a default TTL if `ttl` is not specified.

Reply:
it's enforced here Tony
<img width="552" alt="image" src="https://github.com/user-attachments/assets/f99e9de9-342d-431f-8131-d2e51f26418e">
<img width="765" alt="image" src="https://github.com/user-attachments/assets/8e47b53e-55d0-4687-8fc7-5dccd5044baf">

Reply:
Also, `issueCertFromCa` does not have this because it cannot be called internally. 
<img width="745" alt="image" src="https://github.com/user-attachments/assets/673bb065-cd31-41ed-8d66-58ffa0bdd290">

It can only be invoked via API wherein ttl or notAfter is required